src/client/components/ui/Select/Select.test.tsx

```diff
@@ -3,438 +3,325 @@
 
-			await user.keyboard(KeyBoard.ARROW_UP);
+				const { rerender } = render(<Select optionList={options} value={valueThatCanChange} label={'label'}/>);
 
-			expect(firstOption).toHaveFocus();
-			expect(secondOption).not.toHaveFocus();
-		});
+				valueThatCanChange = '2';
 
-		it('sélectionne une option avec la touche space ET ferme la liste des options', async () => {
-			const user = userEvent.setup();
-			render(
-				<Select
-					placeholder={'Temps de travail'}
-					name="tempsDeTravail"
-					optionList={Offre.TEMPS_DE_TRAVAIL_LIST}
-					label={'Temps de travail'}
-					value=""
-				/>,
-			);
-			const button = screen.getByRole('button', { name: 'Temps de travail' });
-			button.focus();
-			await user.keyboard(KeyBoard.SPACE); // ouverture de la liste des options
-			const optionList = await screen.findByRole('listbox');
-			const firstOption = within(optionList).getAllByRole('option')[0];
-
-			expect(firstOption).toHaveFocus();
-
-			await user.keyboard(KeyBoard.ARROW_DOWN); // focus de la deuxième option
-
-			await user.keyboard(KeyBoard.SPACE); // choix de la deuxième option
-
-			const hiddenInput = await screen.findByTestId('Select-InputHidden');
-			expect(hiddenInput).toHaveValue('tempsPartiel');
-
-			expect(optionList).not.toBeInTheDocument();
+				rerender(<Select optionList={options} value={valueThatCanChange} label={'label'}/>);
+				expect(screen.getByRole('textbox', { hidden: true })).toHaveValue('2');
+			});
 		});
 
-		it('rend le focus au button select après avoir fermé les options', async () => {
+		it('lorsque l‘on donne un name au select simple, on peut récupérer la valeur séléctionnée depuis ce nom', async () => {
 			const user = userEvent.setup();
-			render(
-				<Select
-					placeholder={'Temps de travail'}
-					name="tempsDeTravail"
-					optionList={Offre.TEMPS_DE_TRAVAIL_LIST}
-					label={'Temps de travail'}
-					value=""
-				/>,
-			);
-			const button = screen.getByRole('button', { name: 'Temps de travail' });
-			button.focus();
-			await user.keyboard(KeyBoard.SPACE);
-			const optionList = await screen.findByRole('listbox');
+			const options = [{ libellé: 'options 1', valeur: '1' }, { libellé: 'options 2', valeur: '2' }];
 
-			expect(screen.getByRole('radio', { name: 'Temps plein' })).toBeInTheDocument();
-			expect(screen.getByRole('radio', { name: 'Temps partiel' })).toBeInTheDocument();
-			expect(screen.getByRole('radio', { name: 'Indifférent' })).toBeInTheDocument();
+			render(<form aria-label={'formulaire'}>
+				<Select optionList={options} label={'label'} name={'nomSelect'}/>
+			</form>);
 
-			await user.keyboard(KeyBoard.ESCAPE);
-
-			expect(optionList).not.toBeInTheDocument();
+			await user.tab();
+			await user.keyboard(KeyBoard.ENTER);
+			await user.keyboard(KeyBoard.ARROW_DOWN);
+			await user.keyboard(KeyBoard.SPACE);
 
-			expect(button).toHaveFocus();
+			expect(screen.getByRole('form', { name: 'formulaire' })).toHaveFormValues({ nomSelect: '2' });
```

Comment:
```suggestion
			expect(screen.getByRole('form')).toHaveFormValues({ nomSelect: '2' });
```
Dans ce cas on est en environnement contrôlé, pas besoin de préciser quel formulaire on récupère (et donc pas besoin de donner un `aria-label` au form non plus)

Reply:
pour info si tu ajoutes pas le role form tu ne peux pas recuperer les formvalues a part si tu mets un aria-label